specification/keyvault/Security.KeyVault.EKM/models.tsp

```diff
@@ -26,18 +26,24 @@ model EkmConnection {
   pathPrefix?: string;
 
   /**
-   * The root CA certificate chain that issued the proxy server's certificate. An array of certificates in the certificate chain, each in DER format and base64 encoded.
+   * The root CA certificate chain that issued the proxy server's certificate. An array of certificates in the certificate chain, each in DER format and base64url encoded.
```

Comment:
You have to actually use the `@encoding` decorator. You can't just document it. That won't do anything for our emitters.

Reply:
I tried to change the type to bytes[]. Even with this, it throws `invalid-encode` error. 

<img width="2068" height="168" alt="image" src="https://github.com/user-attachments/assets/26576e95-2d58-465b-b1bc-fe0c99a41383" />